packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/GettingStarted.a/GraphCore/Documentation/Graph.1.doc.mdx

```diff
@@ -1,96 +1,151 @@
-            upperLimit: 250
-        }
+The graph configuration object has a set of properties for the underlying grid of a graph. 
+It hold various properties to configure a grid. 
+
+-   [Valid Graphs](#valid-graphs)
+-   [Configuration Structure](#configuration-structure)
+-   [Configuration Properties](#configuration-structure)
+    -   [clickPassThrough](#clickpassthrough)
+    -   [dimension](#dimension)
+    -   [legendPadding](#legendpadding)
+    -   [ticksCount](#tickscount)
+
+## Valid Graphs
+
+It is required for the following graphs:
+
+- [Bar](../../../bar-graph/about)
+- [Bubble](../../../bubble-graph/about)
+- [Combination](../../../combination/about)
+- [Line](../../../line-graph/about)
+- [Paired Result](../../../paired-result-graph/about)
+- [Scatter](../../../scatter-graph/about)
+
+
+## Configuration Structure
+```js
+const graphConfig = {
+    axis: <Axis object>,
+    bindTo: <string>,
+    allowCalibration: <bool>,
+    bindLegendTo: <string>,
+    clickPassThrough: {
+      datelines: <bool>,
+      eventlines: <bool>,
+    }
+    dateline: <Dateline object>,
+    eventline: <Eventlines object>,
+    legendPadding: { 
+      top: <number>, 
+      bottom: <number>, 
+      left: <number>, 
+      right: <number> 
+    },
+    locale: <locale object>,
+    padding: { 
+      top: <number>, 
+      bottom: <number>, 
+      left: <number>, 
+      right: <number> 
     },
-    showLabel: true,
-    showLegend: true,
-    showShapes: true,
-    showVGrid: true,
-    showHGrid: true
-};
+    pan: {
+      enabled: <bool>
+    },
+    removeContainerPadding: <bool>,
+    showHGrid: <bool>,
+    showLabel: <bool>,
+    showLegend: <bool>,
+    showNoDataText: <bool>,
+    showShapes: <bool>,
+    showLabel: <bool>,
+    showVGrid: <bool>,
+    ticksCount: <number>
+}
 ```
+
+## Configuration Properties
+
+| Property Name          | Expected | Default                                        | Required | Description                                                                           |
+|------------------------|----------|------------------------------------------------|----------|---------------------------------------------------------------------------------------|
+| axis                   | object   | -                                              | yes      | See [Axis](./Axes).                                                                   |
+| bindTo                 | string   | -                                              | yes      | DOM id to bind the graph into.                                                        |
+| allowCalibration       | boolean  | `true`                                         | no       | Toggle to allow calibration to adjust the y axis.                                     |
+| bindLegendTo           | string   | `null`                                         | no       | If DOM id provided, binds legend into that container (Example: `"#legendContainer"`). |
```

Comment:
clickPassthrough link is not working as expected.

Reply:
fixed here: 741f50f900bbdf14d28172159fa3087416114641